RTN-008.tex

```diff
@@ -62,114 +63,113 @@
+Despite this initial success, intense followup of more than a dozen neutron-star-involved mergers in observing run O3 did not yield a credible electromagnetic counterpart \citep[e.g.,][]{2020ApJ...905..145K, 2022ApJ...927...50R, 2022MNRAS.509.3427D}.
+These results suggested that deeper optical surveys are needed in order to identify EM counterparts to these events \citep[e.g.,][]{2020MNRAS.492..863C, 2021MNRAS.504.1294S}.
 
-Indeed, LSST's depth and wide field of view provides great potential for discovery. 
-LSST's strengths will become increasing important in later GW observing runs: 
-as KAGRA and LIGO-India come online, GW spatial localizations will improve to $\sim$tens of square degrees, lessening the need for extremely wide-field instruments to tile major fractions of the sky.  
+Given Rubin's major improvement in depth and survey speed relative to current facilities,  
+this scenario raises substantial community and agency expectations for the Rubin Observatory.
+There is little doubt that Rubin can play a central role in identifying significant samples of EM counterparts to GW sources.
+
+Rubin's strengths will become increasing important in later GW observing runs: 
+as KAGRA and LIGO-India come online, GW spatial localizations will improve to $\sim$tens of square degrees, lessening the need for extremely wide-field instruments to tile major fractions of the sky \citep{2022ApJ...924...54P}.  
 However, as the GW interferometers become more sensitive the GW sources detected will be farther away, making already-faint EM counterparts even fainter.
 
 There will certainly be other EM followup programs active in the Southern Hemisphere, including the new BlackGEM array as well as community programs with DECam.
-However, LSST is capable of making the definitive measurements, delivering high-impact early science along with another major NSF facility in an area of NSF strategic priority\footnote{``Windows on the Universe:'': \url{https://www.nsf.gov/news/special_reports/big_ideas/universe.jsp}}.
+However, Rubin is capable of making the definitive measurements, delivering high-impact early science along with another major NSF facility in an area of NSF strategic priority\footnote{``Windows on the Universe:'': \url{https://www.nsf.gov/news/special_reports/big_ideas/universe.jsp}}.
 
-One challenge remains: the current timeline for future LIGO/Virgo/KAGRA observing runs (Figure \ref{fig:scenarios}) calls for observing run O4 to overlap with the LSST commissioning period and the first year of operations.
+One challenge remains: the current timeline for future LIGO/Virgo/KAGRA observing runs (Figure \ref{fig:scenarios}) calls for observing run O4 to overlap only with the Rubin commissioning period\footnote{See \url{https://dmtn-232.lsst.io/top_milestones.html} for current estimates.}.
```

Comment:
LSSTCam is available at the very most optimistic time January 2024. More likely a few months later. It looks like a clean miss right now. So most of this discussion is to be prepared incase they slip.